client/landing/jetpack-cloud/index.js

```diff
@@ -43,20 +43,13 @@ export default function() {
 			clientRender
 		);
 		page(
-			'/backups/:site/download',
+			'/backups/:site/download/:rewindId',
 			siteSelection,
 			navigation,
 			backupDownload,
 			makeLayout,
 			clientRender
 		);
-		page(
```

Comment:
I recognize the start fo this route from an experiment I did a couple weeks ago, but I really cannot fathom what happened to it since then 😱